lib/src/asset_vector_tile_provider.dart

```diff
@@ -0,0 +1,25 @@
+part of "vector_tile_provider.dart";
```

Comment:
This is the first time using `part` and `part of` in this library. What is the benefit? Why not use files and imports without `part`?

Reply:
I couldn't access the private class _UriProvider otherwise. I could make the class public, but I wasn't sure we wanted users of the plugin to access it.